test/ibmq/test_account_client_v2.py

```diff
@@ -178,6 +178,35 @@ def test_custom_client_app_header(self):
             self.assertNotIn(custom_header,
                              api.client_api.session.headers['X-Qx-Client-Application'])
 
+    def test_list_backends(self):
+        """Test listing backends."""
+        api = self._get_client()
+        provider_backends = {backend.name() for backend
+                             in self.provider.backends()}
+        api_backends = {backend_info['backend_name'] for backend_info
+                        in api.list_backends()}
+
+        self.assertEqual(provider_backends, api_backends)
+
+    @skip('TODO: reenable after api changes')
+    def test_job_cancel(self):
+        """Test canceling a job."""
+        backend_name = 'ibmq_qasm_simulator'
+        backend = self.provider.get_backend(backend_name)
+        circuit = transpile(self.qc1, backend, seed_transpiler=self.seed)
+        qobj = assemble(circuit, backend, shots=1)
+
+        api = backend._api
+        job = backend.run(qobj)
+        job_id = job.job_id()
+
+        try:
+            api.job_cancel(job_id)
+        except RequestsApiError as ex:
+            # TODO: rewrite using assert
+            if all(err not in str(ex) for err in ['JOB_NOT_RUNNING', 'JOB_NOT_CANCELLED']):
```

Comment:
Added a `TODO` here in the hopes of being able to use asserts eventually when we reenable the test :grimacing: